.gitignore

```diff
@@ -66,3 +66,8 @@ libstandaloneengine.a
 tests/string
 tests/config
 
+# compdb
+.cache/
+compile_commands.json
+
+
```

Comment:
I suggest deleting the extra empty lines at the end of the file. (But keep the LF at the end of the last line)

Reply:
Its done, thanks for your advice